src/stream.rs

```diff
@@ -1,14 +1,13 @@
-use std::io::{Read, Seek};
```

Comment:
I see the imports have been sorted and newlines removed. 

I always keep imports separated into three sections each with a empty line in between. std imports at the top, then those from other libs finally crate local imports at the bottom. 

Its something I have copied from examples and other projects such as Tokio. Though the order seems to change somewhat from project to project. 

We do not have that in the rodio contributors guide atm. I would be for adding it and keeping the current import formatting. What is your preference here?

Reply:
This change by `cargo fmt`.